readme.md

```
@@ -104,6 +104,22 @@ The website consists of the following key sections:
 
 The project uses Bootstrap’s grid system to ensure the website is fully responsive across different screen sizes. Additional media queries are included for further fine-tuning the layout for mobile devices.
 
+![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/rainbow.png)
+
+
+### 💗 Contributors
+
+<a href="https://github.com/GarimaSingh010/Resum-Resume/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=GarimaSingh0109/Resum-Resume" />
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)